tests/features/person.feature

```diff
@@ -0,0 +1,172 @@
+@api
+Feature: Person content creation
+  In order to have "Person" on the site
+  As an editor
+  I need to be able to create and see oe_person items
+
+  @javascript
+  Scenario: Creation of a Person content through the UI.
+    Given I am logged in as a user with the "create oe_person content, access content, edit own oe_person content, view published skos concept entities, manage corporate content entities" permission
+    And the following images:
+      | name          | file           | alt                |
+      | Image 1       | example_1.jpeg | Alternative text 1 |
+      | Image 2       | example_1.jpeg | Alternative text 2 |
+      | Contact image | example_1.jpeg | Alternative text 4 |
+    And the following document:
+      | name          | file         |
+      | My Document 1 | sample.pdf   |
+      | My Document 2 | document.pdf |
+    And the following "Organisation" Content entity:
+      | Title             | Organisation demo page         |
+      | Introduction      | Organisation introduction text |
+      | Acronym           | Organisation acronym           |
+      | Body text         | Organisation body text         |
+      | Organisation type | EU organisation                |
+      | EU organisation   | Directorate-General for Budget |
+    # Create a "person" content, mandatory fields first.
+    And I visit "the Person creation page"
```

Comment:
```suggestion
    When I visit "the Person creation page"
```

Structure should be **Given** And And And **When** And And And **Then** And And And. All test steps should be reviewed and fixed. There are lots of places to fix.

Reply:
yes good point, implemented